src/components/form/templates/field.ts

```diff
@@ -1,12 +1,20 @@
 import React from 'react';
+import { FormLayoutOptions } from '../form.types';
 
 export const FieldTemplate = (props) => {
     const { classNames, children } = props;
+    const layout: FormLayoutOptions = props.schema.lime?.layout;
+    const style: any = {};
+
+    if (layout?.fullWidth) {
+        style.gridColumn = '1/-1';
```

Comment:
Had to google this one 😄

If I understand MDN's description properly, this means the item will start at the first column and end at the last column (precisely as expected). Or, more technically, the `1` means it should start at the first column as counted from the start, and the `-1` means it should end at the first column from the end… 😅

Reply:
Yeah I think that's right :slightly_smiling_face:  It's not the most intuitive syntax and that's why I was hesitant to have `span` on the interface.
Maybe we could have `span: number | 'all'` or something like that though because I'm not 100% happy about the `fullWidth: boolean` either :disappointed:

Reply:
Hmm… yeah, that's a pretty neat idea, if we can make that work.

On the other hand, do we really need it right now? Perhaps we could do `span: 'all'` for now? That would just work exactly like the `fullWidth: boolean` for now, but it would be ready to grow into `span: number | 'all'`…

Reply:
No we don't really need it right now, but we also want to make sure that it will be somewhat compatible with future changes so we don't have to make it breaking right away :smile:

Reply:
But if we make it `span?: 'all'` now, changing it to `span?: number | 'all'` later shouldn't be breaking I think? That change should only be breaking to the implementer, but not to the consumer (I think?), and since the implementation is in this same package, we should be able to make that change without it being breaking.

That way we don't have to spend time making number values work right now 🙂

Reply:
Yeah let's go with the `span` option instead, it feels more natural and I think we should be able to implement numbers without major issues in the future :slightly_smiling_face:

Reply:
😍